### PR TITLE
Released 4.0.1-beta.1

### PR DESCRIPTION
**Pull request by ayushbansal-so**
merged master
released new version

### CODE REVIEW

**Comment by oh-frontend** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **so-sdhawan**
You can assign the PR to them by writing `/assign @so-sdhawan` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/team/technocrats/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by sohimanshu — APPROVED** [review]

**Comment by oh-frontend** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[sohimanshu](https://github.com/scheduleonce/knowledgeowl-angular/pull/71#pullrequestreview-927457948 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/team/technocrats/OWNERS)~~ [sohimanshu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oh-frontend** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[sohimanshu](https://github.com/scheduleonce/knowledgeowl-angular/pull/71#pullrequestreview-927457948 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/team/technocrats/OWNERS)~~ [sohimanshu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
